lib/tasks/upgrade.rake

```diff
@@ -369,4 +369,18 @@ namespace :upgrade do
     Rake::Task['upgrade:remove_duplicated_non_customised_template_versions'].execute
     Rake::Task['upgrade:remove_duplicated_customised_template_versions'].execute
   end
+
+  desc "Remove admin preferences"
+  task remove_admin_preferences: :environment do
```

Comment:
You should call this from the `v1_1_2` task so that users only need to run the one scripts